### ISSUE TITLE
Narrator announces all the options of the menu when user navigates to the first option

### ISSUE THREAD

**Issue by anjali2408** [OfficeDev/office-ui-fabric-react#5264]
### Bug Report

- __Package version(s)__: 5.76.0

#### Priorities and help requested (not applicable if asking question):

Requested priority: (Blocking, High, Normal, Low) : Normal

Products/sites affected: (if applicable): Power BI Visio Visual

#### Describe the issue:

Command bar used with only far items defined. When the user clicks on any item in the commandbar to open a contextual menu , and navigates to the first option in the menu, Narrator announces about all the options in the menu. Dropdowns also have this issue

#### Actual behavior:

Command bar used with only far items defined. When the user clicks on any item in the commandbar to open a contextual menu , and navigates to the first option in the menu, Narrator announces about all the options in the menu.

#### Expected behavior:

Narrator should only announce about the focused option.
User Impact:
Visually Impaired users may get confused with the current announcement.

**Comment by micahgodbolt** [comment]
@anjali2408 the commandBar has been completely refactored in our 6.0 release. We'll only be doing critical fixes to the 5.x version.

But Dropdown on the other hand hasn't changed dramatically. I'll change the title over to that and focus on that problem. They might be related. If it's a super simple fix I may be able to fix it in both.

**Comment by micahgodbolt** [comment]
just tried it on the current website and had zero narrator problems. after clicking it only states the current item, both for dropdown and for commandbar.

Can you confirm this on our website as well? It could be a bug in the version you're running (which is a bit outdated)

**Comment by cliffkoh** [comment]
@micahgodbolt the wrong component is tagged - you really mean commandbar and not dropdown component right?

**Comment by cliffkoh** [comment]
@anjali2408 We will close this issue as @micahgodbolt is unable to repro. Please feel free to re-open if you disagree with the resolution.
